### PR TITLE
Feat: #38

### PR DESCRIPTION
**Pull request by View12138**

### CODE REVIEW

**Comment by CollinAlpert** [comment]
Thanks for this, it's a good idea! Can you make sure the tests pass before I review the PR?

File: Lombok.NET/ConstructorGenerators/RequiredArgsConstructorGenerator.cs [hunk 1]
```diff
@@ -48,43 +48,43 @@ protected override (SyntaxKind modifier, ParameterListSyntax constructorParamete
...
-					.OfType<FieldDeclarationSyntax>()
-					.Where(IsFieldRequired)
-					.Where(static p => !p.Modifiers.Any(SyntaxKind.StaticKeyword))
-					.Where(accessType)
-					.ToList();
-				if (fields.Count == 0)
 				{
-					return (modifier, ParameterList(), Block());
-				}
+					var fields = typeDeclaration.Members
```
Comment (CollinAlpert) [inline Lombok.NET/ConstructorGenerators/RequiredArgsConstructorGenerator.cs]: These seem to be only whitespace changes, please revert.

**Comment by View12138** [comment]
I'm sure that the PR code can pass the test, because I can't run your test project locally, so I wrote another test project myself to test the code I submitted, and the new code in the issue is generated by my local test project

**Comment by View12138** [comment]
I'd also like to share with you my test scenario, which can be used to get into debug mode using VS's debug test method without having to attach to a process, which I think might be easier to write code and test projects

**Review by CollinAlpert — CHANGES_REQUESTED** [review]

**Comment by CollinAlpert** [comment]
Please let me know why you can't run the test project locally, this is something I would definitely need to fix.

Keep in mind you need to run the tests while in Release mode so the debugger does not wait for an attached process: https://github.com/CollinAlpert/Lombok.NET?tab=readme-ov-file#building-and-debugging

File: Lombok.NET/ConstructorGenerators/RequiredArgsConstructorGenerator.cs (hunk 1 above)
Comment (View12138) [reply Lombok.NET/ConstructorGenerators/RequiredArgsConstructorGenerator.cs]: I've already fixed it.

**Comment by View12138** [comment]
This is my [test project](https://github.com/View12138/Lombok.NET/blob/view-test-project/Lombok.NET.Tests/ConstructorGeneratorsTests.cs)

**Comment by CollinAlpert** [comment]
Sorry about the delay, I've had a busy week but will get to this on the weekend the latest.

**Comment by CollinAlpert** [comment]
Looks good! I will look into using [Verify](https://github.com/VerifyTests/Verify) for tests in the future.

**Pull request merged**